scripts/release/release_info.py

```diff
@@ -1,90 +1,149 @@
+    DATABASE_IMAGE,  # mongodb-kubernetes-database
+    READINESS_PROBE_IMAGE,  # mongodb-kubernetes-readinessprobe
 ]
 
-# TODO: this is dummy version, to be replaced with actual versioning logic https://docs.google.com/document/d/1eJ8iKsI0libbpcJakGjxcPfbrTn8lmcZDbQH1UqMR_g/edit?tab=t.45ig7xr3e3w4#bookmark=id.748ik8snxcyl
-DUMMY_VERSION = "dummy_version"
-
 
-def create_release_info_json() -> str:
+def create_release_info_json(operator_version: str) -> str:
     build_info = load_build_info(scenario=BuildScenario.RELEASE)
 
-    release_info_json = convert_to_release_info_json(build_info)
+    release_info_json = convert_to_release_info_json(build_info, operator_version)
 
     return json.dumps(release_info_json, indent=2)
 
 
-def convert_to_release_info_json(build_info: BuildInfo) -> dict:
-    output = {
+def convert_to_release_info_json(build_info: BuildInfo, operator_version: str) -> dict:
+    release_json_data = os.path.join(os.getcwd(), "release.json")
+    with open(release_json_data, "r") as fd:
+        release_data = json.load(fd)
+
+    release_info_output = {
         "images": {},
-        "binaries": {},
-        "helm-charts": {},
     }
     # Filter (and order) images to include only those relevant for release info
     images = {name: build_info.images[name] for name in RELEASE_INFO_IMAGES_ORDERED}
 
     for name, image in images.items():
-        output["images"][name] = {
-            "repositories": image.repositories,
-            "platforms": image.platforms,
-            "version": DUMMY_VERSION,
-        }
-
-    for name, binary in build_info.binaries.items():
-        output["binaries"][name] = {
-            "platforms": binary.platforms,
-            "version": DUMMY_VERSION,
-        }
-
-    for name, chart in build_info.helm_charts.items():
-        output["helm-charts"][name] = {
-            "registry": chart.registry,
-            "repository": chart.repository,
-            "version": DUMMY_VERSION,
-        }
+        add_image_info(release_info_output, name, image.repositories, image.platforms, operator_version)
+
+    # add OPS manager image info
+    om_build_info = build_info.images[OPS_MANAGER_IMAGE]
+    add_image_info(
+        release_info_output,
+        OPS_MANAGER_IMAGE,
+        om_build_info.repositories,
+        om_build_info.platforms,
+        latest_om_version(release_data),
+    )
+
+    # add agent image info
+    agent_build_info = build_info.images[AGENT_IMAGE]
+    add_image_info(
+        release_info_output,
+        AGENT_IMAGE,
+        agent_build_info.repositories,
+        agent_build_info.platforms,
+        latest_agent_version(release_data),
+    )
+
+    # add upgrade hook image info
+    upgradehook_build_info = build_info.images[UPGRADE_HOOK_IMAGE]
+    add_image_info(
+        release_info_output,
+        UPGRADE_HOOK_IMAGE,
+        upgradehook_build_info.repositories,
+        upgradehook_build_info.platforms,
+        latest_upgrade_hook_version(release_data),
```

Comment:
I see `upgradehook` but `readiness probe` is still missing. 

This should show up in the tests as well, you might need to update it.

Reply:
I have added readiness probe image as well, I didn't notice that it's version is also mentioned in the release.json.

Which tests are you talking about here?

Reply:
It looks like we are better off without the `release_info_test.py`. And the reason is, if we want to test the `release_info.py`, the test file is not just going to test logic/functions, it will have actual logic (that the file that we are testing has), to figure things out. 
For example, it (the test file) will have to figure a lot of versions out from `release.json` and most importantly it will have to figure out the digest of the images and then use that in json that we are going to test against. 
So, in my opinion we should get rid of release_info_test.py, what do you think?

Reply:
>The problem with this approach is, we will have to update this test file as well along with release.json, when new versions of OM/Agent are released.

Actually we won't need to update `test_release.json` ever, this is the main benefit of having separate file, to test only the logic, not the actual contents of the `release.json` file. We don't care about new versions released and I don't understand why we would need to update `test_release.json` when this happens. Can you explain why you think so?